src/vs/editor/contrib/dropOrPasteInto/browser/copyPasteController.ts

```diff
@@ -283,6 +283,12 @@ export class CopyPasteController extends Disposable implements IEditorContributi
 					return;
 				}
 
+				// If the only edit returned is a text edit, use the default paste handler
```

Comment:
```suggestion
				// If the only edit returned is a text edit, use the default paste handler.
```
This is just for consistency, like on [line 272](https://github.com/microsoft/vscode/pull/192220/files#diff-3558a9a40fa1a73d2dc9f1b6c00c184581eaa73ff926625b390ab3bf6e116ea6R272). Feel free to ignore this if you don't think it's helpful.